### PR TITLE
[Left side panel] UI bugs of Left side panel

### PR DESCRIPTION
**Pull request by MiteshDarda**
## Description

Fixed [Left side panel] UI bugs of Left side panel
- Design was not up to date with figma which is now fixed
- Add Collection Feature was not present in workspace Add List which is now fixed

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: src/packages/@app/repositories/environment.repository.ts [hunk 1]
```diff
@@ -99,4 +99,23 @@ export class EnvironmentRepository {
...
       .remove();
   };
+
+  /**
+   * Retrieves an array of test flow documents by their workspace ID.
+   *
+   * @param  _workspaceId - The unique identifier of the workspace to filter the test flows.
+   * @returns A promise that resolves to an array of environment documents associated with the given workspace ID.
+   */
+  public getEnvironmentByWorkspaceId = async (
```
Comment (itsmdasifraza) [inline src/packages/@app/repositories/environment.repository.ts:109]: update comment please

Comment (itsmdasifraza) [reply src/packages/@app/repositories/environment.repository.ts:109]: and remove consoles

File: src/packages/@workspaces/features/workspace-actions/layout/WorkspaceActions.svelte [hunk 2]
```diff
@@ -297,6 +298,16 @@
...
+        {
+          name: "Add Environment",
+          icon: StackIcon,
+          iconColor: "var(--icon-secondary-130)",
+          iconSize: "15px",
+          onclick: () => {
+            isExpandEnvironment = true;
+            onCreateEnvironment();
+          },
+        },
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/workspace-actions/layout/WorkspaceActions.svelte:310]: there will we one more array for guest user

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**